src/Autofac.Integration.AspNetCore.Multitenant/Autofac.Integration.AspNetCore.Multitenant.csproj

```
@@ -36,12 +38,12 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.1.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp3.1'">
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp3.1' or $(TargetFramework) == 'net5.0'">
```

```suggestion
  <ItemGroup Condition="$(TargetFramework) != 'netstandard2.0'">
```
This will make it easier to add new ones later, and if we remove `netstandard2.0` at some point and forget this things will still work.

---

Applied.